ext/data/recommended-dictionaries.json

```diff
@@ -0,0 +1,15 @@
+{
+    "ja": {
+        "terms": [
+            {"name": "Jitendex", "url": "https://github.com/stephenmk/stephenmk.github.io/releases/latest/download/jitendex-yomitan.zip"}
+        ],
+        "kanji": [
+            {"name": "Kanjidic English", "url": "https://github.com/themoeway/jmdict-yomitan/releases/latest/download/KANJIDIC_english.zip"}
+        ],
+        "frequency": [
+            {"name": "BCCWJ-LUW", "url": "https://github.com/toasted-nutbread/yomichan-bccwj-frequency-dictionary/releases/download/1.0.1/BCCWJ-LUW.zip"}
```

Comment:
```suggestion
            {"name": "BCCWJ", "url": "https://github.com/Kuuuube/yomitan-dictionaries/raw/main/dictionaries/BCCWJ_SUW_LUW_combined.zip"}
```

Reply:
Made a release on the repo for this and linked there instead since I do not want to be guaranteeing there will never be a change in repo structure that could break that link.